src/audio_core/audio_renderer.cpp

```diff
@@ -29,10 +29,9 @@ namespace {
                                        (static_cast<float>(r_channel) * r_mix_amount)));
 }
 
-[[nodiscard]] static constexpr std::tuple<s16, s16> Mix6To2(s16 fl_channel, s16 fr_channel,
-                                                            s16 fc_channel,
-                                                            [[maybe_unused]] s16 lf_channel,
-                                                            s16 bl_channel, s16 br_channel) {
+[[maybe_unused]] [[nodiscard]] static constexpr std::tuple<s16, s16> Mix6To2(
```

Comment:
As Breadfish suggested on discord
```suggestion
[[maybe_unused, nodiscard]] static constexpr std::tuple<s16, s16> Mix6To2(
```